seed/authentication.py

```diff
@@ -16,17 +18,28 @@ class SEEDAuthentication(authentication.BaseAuthentication):
     """
 
     def authenticate(self, request):
-        auth_header = request.META.get('HTTP_AUTHORIZATION')
+        auth_header = request.META.get('Authorization')
+
+        if not auth_header:
+            auth_header = request.META.get('HTTP_AUTHORIZATION')
 
         if not auth_header:
             return None
 
         try:
-            from seed.landing.models import SEEDUser as User
+            if not auth_header.startswith('Basic'):
+                raise exceptions.AuthenticationFailed(
+                    "Only Basic HTTP_AUTHORIZATION is supported"
+                )
+
+            auth_header = auth_header.split()[1]
+            auth_header = base64.urlsafe_b64decode(auth_header)
```

Comment:
Since these tests pass on my local machine, I am reasonably certain that the tests will work on travis once the issue of travis not using the updated requirements for this branch (specifically getting travis to install django-filter) is resolved.